runtime/docs/CHANGELOG.md

```diff
@@ -7,10 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.0.0.9] - 2023-01-23
+## [0.0.0.10] - 2023-01-23
 
 - Updated README.md with information about node profiling
 
+## [0.0.0.9] - 2023-01-20
+
+- move ConnectionType enum into its own package to avoid a cyclic import between configs and defaults packages (i.e. configs -> defaults -> configs) in the resulting, generated go package
+- update makefile protogen_local target to build additional proto file and include it in the import path for runtime/configs/proto/p2p_config.proto
+- replace `P2PConfig#IsEmptyConnectionType` bool with `P2PConfig#ConnectionType` enum
+- replace `DefaultP2PIsEmptyConnectionType` bool with `DefaultP2PConnectionType` enum
+
```

Comment:
I saw that you posted #461 and will check it out in the coming days. Until we discuss it as a team and settle on the new pattern, let's use the current approach to avoid discrepancies.

```suggestion
## [0.0.0.10] - 2023-01-25

- move ConnectionType enum into its own package to avoid a cyclic import between configs and defaults packages (i.e. configs -> defaults -> configs) in the resulting, generated go package
- update makefile protogen_local target to build additional proto file and include it in the import path for runtime/configs/proto/p2p_config.proto
- replace `P2PConfig#IsEmptyConnectionType` bool with `P2PConfig#ConnectionType` enum
- replace `DefaultP2PIsEmptyConnectionType` bool with `DefaultP2PConnectionType` enum

## [0.0.0.9] - 2023-01-23

- Updated README.md with information about node profiling
```

For reference, my workflow, for now, has been:

```bash
gco main
git pull
gco feature_branch
git merge main
// resolve conflicts
git commit -am "Merge with main"
git push
```